plugins/topology/src/common/components/Status.tsx

```diff
@@ -63,13 +65,20 @@ const Status = ({ status, noTooltip, iconOnly }: StatusProps) => {
       return (
         <StatusIconAndText
           {...statusProps}
-          icon={<RedExclamationCircleIcon />}
+          icon={
+            <ExclamationCircleIcon className="bs-topology-icons__red-exclamation-icon" />
```

Comment:
@debsmita1 since you are using pf v5 color palette for icons in tekton PR - https://github.com/janus-idp/backstage-plugins/pull/734/files#diff-3b7c01df880a9b9228daac5520b8a879b90a16e9ca5aab39dc751f23716344acR102 so to keep it consistent we should use pf v5 color palette here as well.

Reply:
Yes, these classes are using the pf5 color palette

```
Status.css


.bs-topology-status__green-check-icon {
  fill: var(--pf-v5-global--palette--green-500) !important;
}

.bs-topology-status__red-exclamation-icon {
  fill: var(--pf-v5-global--danger-color--100) !important;
}